.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Report a bug and help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1.
+2.
+3.
+
+**Environment**:
+- multus-dynamic-networks-controller version: N/A
+- Kubernetes version (use `kubectl version`): N/A
+- Network-attachment-definition: N/A
+- OS (e.g. from /etc/os-release): N/A
+- Kernel (e.g. `uname -a`): N/A
```

Comment:
Is this useful? Perhaps we should ask for the runtime instead?

Reply:
Added the question about the runtime, good catch.

I still think the kernel version is useful; let's say a user opens a bug here, and after some debugging I figure out the real issue is elsewhere - I will need to open a bug on another repo that will quite probably ask me this question (for instance multus).

Reply:
Fair enough